files/image_config/monit/memory_checker

```diff
@@ -59,6 +69,81 @@ def get_command_result(command):
 
     return command_stdout.strip()
 
+def get_container_id(container_name):
+    """Gets full container ID of the specified container
+    Args:
+        container_name: A string indicates the name of specified container.
+    Returns:
+        container_id: A string indicates the full ID of specified container.
+    """
+    container_id = ""
+
+    get_container_info_cmd = "docker ps --no-trunc"
```

Comment:
Maybe we can use this command to filter specific container name?

get_container_info_cmd = "docker ps --no-trunc --filter name={}".format(container_name)

Reply:
Good point. done.